mev_inspect/aave_liquidations.py

```diff
@@ -0,0 +1,128 @@
+from typing import List
+
+from mev_inspect.schemas.classified_traces import (
+    ClassifiedTrace,
+    DecodedCallTrace,
+    Classification,
+)
+from mev_inspect.schemas.liquidations import Liquidation
+from mev_inspect.schemas.transfers import ERC20Transfer
+
+contract_addresses = [
```

Comment:
which contract addresses? what do these represent?

also for constants the convention is capitals `CONTRACT_ADDRESSES`

Reply:
the addresses of the smart contracts through which the liquidation is made

Reply:
maybe `LIQUIDATION_CONTRACT_ADDRESSES`?

Reply:
done